docs/source/crs/projections.rst

```diff
@@ -384,3 +384,18 @@ SouthPolarStereo
     ax.gridlines()
 
 
+Sinusoidal
```

Comment:
This does not belong at this point; you need to run `make projection` in the `docs` directory to get them in the correct sorted order.